src/highlight/HighlightManager.tsx

```diff
@@ -35,8 +35,8 @@ export default class HighlightManager implements BaseManager {
         rootLayerEl.dataset.testid = 'ba-Layer--highlight';
         rootLayerEl.setAttribute('data-resin-feature', 'annotations');
 
-        // Insert the new layer element immediately after the reference element
-        return parentEl.insertBefore(rootLayerEl, referenceEl.nextSibling);
+        // Append the new layer node to the end of the list of children of the parent node
+        return parentEl.appendChild(rootLayerEl);
```

Comment:
This is going to remove some flexibility we have today where annotations can be interleaved with layers of the viewer. We may need that functionality, particularly for highlight annotations that may need to appear behind the text layer. Maybe we should hold off on this for now.

Reply:
I see. I'm going to close this PR for now. We can re-open it if we need it in the future.